main/lsp/QueryCollector.cc

```diff
@@ -11,7 +11,11 @@ u2 getQueryResponseTypeSpecificity(const core::lsp::QueryResponse &q) {
         return 8;
     } else if (q.isDefinition()) {
         return 7;
-    } else if (q.isSend()) {
+    } else if (auto send = q.isSend()) {
+        if (send->callerSideName == core::Names::defineTopClassOrModule()) {
+            // LSP doesn't need to know about this; sort it to the end.
+            return 1;
+        }
```

Comment:
This seems out of place; there are a bunch of synthetic method names that get introduced throughout Sorbet's pipeline. What caused us to need to single this one out? (Is this fixing a bug that was always there regardless of packaging, but that you happened to notice and fix now?)

Reply:
Given the way the tests change, there might be a test for this, but I would love if you could confirm whether this shows up in our test suite somewhere or not.

Reply:
iirc, hover on an `import Foo` statement would display 'Void' because a position query returned this send.

I asked @elliottt what this is -- we could either mark it with a 0-sized loc to ignore it entirely (which is what I believe we do with other synthetic things) or I could filter it out manually -- and we decided on the latter since I guess the loc is used for the compiler.

Reply:
I don't think we do use the loc for the compiler, just its place in the tree.

Reply:
Cool, let me see if I can zero-size it and still pass the tests.

Reply:
It works! Fixed.